_pages/docs/gradletasks.md

```diff
@@ -231,3 +231,16 @@ gradle hubDeleteJobs \
 #### Parameters
 ##### Required
 - **jobIds** - a comma-separated list of job IDs to delete
+
+### hubExportJobs
+Export job records and the traces that go with them. Does not affect the content of the staging or final databases.
+
+<pre class="cmdline">
+gradle hubExportJobs \
+  -PjobIds=list-of-ids -Pfilename=export.zip
```

Comment:
It's possible to call w/o the jobIds, right? If so, can you mention this? Tell me that it exports all.